app/timetopaytaxpayer/sa/PaymentStubsProtectedConnector.scala

```diff
@@ -26,12 +26,12 @@ import uk.gov.hmrc.play.bootstrap.http.HttpClient
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class SaConnector @Inject() (
+class PaymentStubsProtectedConnector @Inject()(
```

Comment:
The name and the config should be left as they were. The stub means that the production dependant system is stubbed on test environments. The application should not now if it runs against stubs. 
Please revert these renamings

Reply:
Was about to say the same ... override SA settings with that of the stub.
In general we would not right production code pointing to a stub ...we override code pointing to a service to point to a stub ... there should not be a situation where we point to a stub in all envs